src/extensions/common/CanvasWidget/CanvasPainter.ts

```diff
@@ -34,7 +34,7 @@ interface TextAlignmentConfig {
 }
 
 const rotateRect = (r: RectangularRegion) => {
-    return {xmin: -r.ymax, xmax: -r.ymin, ymin: r.xmax, ymax: r.xmin}
+    return {xmin: -r.ymax, xmax: -r.ymin, ymin: r.xmin, ymax: r.xmax}
```

Comment:
Okay. So this corresponds to a rotation of 90 degrees (counterclockwise) around the origin.

I did a few sketches and it looks to check out.